explorer/scan/local_scanner.go

```diff
@@ -561,13 +591,16 @@ func (s *localAssetScanner) runQueryPack() (*AssetReport, error) {
 	var conductor explorer.QueryConductor = s.services
 
 	log.Debug().Str("asset", s.job.Asset.Mrn).Msg("client> request bundle for asset")
-	assetBundle, err := hub.GetBundle(s.job.Ctx, &explorer.Mrn{Mrn: s.job.Asset.Mrn})
-	if err != nil {
-		return nil, err
+	// If we run in debug mode, download the asset bundle and dump it to disk
+	if val, ok := os.LookupEnv("DEBUG"); ok && (val == "1" || val == "true") {
```

Comment:
we only dump this if the env var `DEBUG=1` is set. I think `log.Debug().Enabled()` is also `true` when we just have `--log-level debug`. I don't think we want to have these files in that case. 

That same condition exists in cnspec for dumping the bundle there too. I just used the same approach

Reply:
temporarily ok, but we need to extract this into a function. Otherwise it will be hard to change that in the future https://github.com/mondoohq/cnquery/blob/main/logger/debug.go#L77